guides/v2.2/install-gde/composer.md

```diff
@@ -18,10 +18,10 @@ functional_areas:
 
 Before you continue, you must do the following:
 
--  Set up a server that meets our [system requirements][]
--  Create the [Magento file system owner][]
--  [Install Composer][]{:target="_blank"}
--  Obtain [authentication keys][] for the Magento code repository
+- Set up a server that meets our [system requirements][]
```

Comment:
The linter requires two spaces after the dash in an unordered list.

Reply:
Thanks @keharper. It seems that I need to update my local configuration for markdownlint.